2021/yearbook.html

```diff
@@ -32,6 +32,13 @@
     <h2>2021 Software Development Engineer Bootcamp</h2>
 
     <div class="yearbook">
+           <a>
+       <a href="https://mita-cpu.github.io/" target="_blank" />
+      <figure>
+          <img src="images/my-picture.jpg" alt="Mita" />
```

Comment:
Please create a `mita.jpg` file and restore other deleted files `nothemba.jpg`, `anthony.jpg`

Reply:
i have created a mita.jpg